pages/linux/swaks.md

```diff
@@ -0,0 +1,24 @@
+# swaks
+
+> Swiss Army Knife SMTP, the all-purpose SMTP transaction tester.
+> More information: <https://github.com/jetmore/swaks/blob/0/doc/base.pod>.
+
+- Deliver a standard test email to user@example.com on port 25 of test-server.example.net:
+
+`swaks --to {{user@example.com}} --server {{test-server.example.net}}`
+
+- Deliver a standard test email, requiring CRAM-MD5 authentication as user me@example.com. An "X-Test" header will be added to the email body. The authentication password will be prompted for if it cannot be obtained from your .netrc file:
+
+`swaks --to {{user@example.com}} --from {{me@example.com}} --auth {{CRAM-MD5}} --auth-user {{me@example.com}} --header-X-Test {{"test email"}}`
+
+- Test a virus scanner using EICAR in an attachment. Don't show the message DATA part:
+
+`swaks -t {{user@example.com}} --attach - --server {{test-server.example.com}} --suppress-data {{/path/to/eicar.txt}}`
```

Comment:
Should be absolute path?
```suggestion
`swaks -t {{user@example.com}} --attach - --server {{test-server.example.com}} --suppress-data {{path/to/eicar.txt}}`
```

Reply:
Should there be an extra `-` between --attach and --server?